universal-login-contracts/test/fixtures/basicIdentity.js

```diff
@@ -1,17 +1,20 @@
 import ERC1077ApprovalScheme from '../../build/ERC1077';
 import MockToken from '../../build/MockToken';
 import MockContract from '../../build/MockContract';
-import {utils} from 'ethers';
+import {utils, Wallet} from 'ethers';
 import {deployContract} from 'ethereum-waffle';
 import {addressToBytes32} from '../utils';
-import {OPERATION_CALL} from '../../lib/consts';
+import {OPERATION_CALL, ACTION_KEY} from '../../lib/consts';
 import DEFAULT_PAYMENT_OPTIONS from '../../lib/defaultPaymentOptions';
 
 const {parseEther} = utils;
 const {gasPrice, gasLimit} = DEFAULT_PAYMENT_OPTIONS;
 
 export default async function basicIdentity(provider, [, , , , , , , , , wallet]) {
   const publicKey = addressToBytes32(wallet.address);
+  const actionWallet = Wallet.createRandom();
```

Comment:
👍 A nice alternative way of creating keys. We should use that instead of getWallets, whenever we want a key without balance on it.